test/integration/api_docs/services_controller_test.rb

```diff
@@ -62,6 +62,7 @@ def test_index_and_show
 
         assert_response :success
         assert show_result.has_key?('paths')
+        assert_equal show_result["openapi"], "3.0.1"
```

Comment:
Do you think you could assert for a regular expression that matches `/^3\.[0-9]+\.[0-9]+$/` ?

Reply:
but is that really needed to make it so complex when we can directly check the version

Reply:
I think it's good to have, so the test still works for future 3.x.x versions of the spec. The implementation is in fact trivial: only calling `assert_match` instead of `assert_equal`. The hardest part is to get to the regular expression itself, but I'm already providing it.

On the other hand, remember that in Minitest assertions the expected value comes first, so even if you want to use `assert_equal`, it should be `assert_equal "3.0.1", show_result["openapi"]`

Reply:
Okay let me add and update this test

Reply:
added